plugins/ufm_log_analyzer_plugin/src/loganalyze/log_analyzers/ibdiagnet2_port_counters_analyzer.py

```diff
@@ -52,11 +53,16 @@ def __init__(
         self._first_timestamp_of_logs = None
         self._last_timestamp_of_logs = None
 
-    def get_collectx_versions(self):
+    def save_collectx_versions(self):
         unique_collectx_versions = self._log_data_sorted[
             self._log_data_sorted["type"] == "collectx_version"
         ]["data"].unique()
-        return unique_collectx_versions
+        self._dataframes_for_pdf.append(
+            (
+                f"{self.telemetry_type} telemetry collectx versions",
```

Comment:
collectx versions found in {self.telemetry_type} telemetry log

Reply:
fixed